src/main/java/nl/tudelft/ti2206/group9/gui/skins/AbstractSkin.java

```diff
@@ -0,0 +1,58 @@
+package nl.tudelft.ti2206.group9.gui.skins;
+
+import javafx.scene.paint.PhongMaterial;
+import nl.tudelft.ti2206.group9.gui.Style;
+
+/**
+ * @author Maikel on 08/10/2015.
+ */
+public abstract class AbstractSkin {
```

Comment:
Why is this class abstract, while there are no abstract methods defined?

<review,logic,interface